sdk/remoterendering/azure-mixedreality-remoterendering/samples/remote_rendering_client_sample.py

```diff
@@ -128,7 +131,10 @@ def perform_asset_conversion():
         print("conversion with id:", conversion_id, "created. Waiting for completion.")
         conversion = conversion_poller.result()
         print("conversion with id:", conversion_id, "finished with result:", conversion.status)
-        print(conversion.output.asset_uri)
+        if conversion.output != None:
+            print(cast(AssetConversionOutput, conversion.output).asset_uri)
```

Comment:
What is the reason for the cast? We should not make our users cast in their code, if possible.